packages/core/http/core-http-router-server-internal/src/versioned_router/core_versioned_route.test.ts

```diff
@@ -86,7 +86,7 @@ describe('Versioned route', () => {
     );
   });
 
-  it('only allows versions that are numbers greater than 0', () => {
+  it('only allows versions that are numbers greater than 0 for internal APIs', () => {
```

Comment:
Still enforce monotonic increasing numbers for internal APIs (should we need them)